source/imaer-gml/src/main/java/nl/overheid/aerius/gml/base/source/mobile/v31/GML2OffRoad.java

```diff
@@ -94,8 +96,10 @@ private OffRoadMobileSource convert(final IsGmlCustomOffRoadMobileSource customM
     final CustomOffRoadMobileSource customVehicleEmissionValues = new CustomOffRoadMobileSource();
     customVehicleEmissionValues.setDescription(customMobileSource.getDescription());
 
-    customVehicleEmissionValues.setCharacteristics(gml2SourceCharacteristics.fromGML(customMobileSource.getCharacteristics(),
-        null, null));
+    final OPSSourceCharacteristics gmlCharacteristics = (OPSSourceCharacteristics) gml2SourceCharacteristics.fromGML(
+        customMobileSource.getCharacteristics(), null, null);
+    gmlCharacteristics.setDiurnalVariation(DiurnalVariation.INDUSTRIAL_ACTIVITY);
```

Comment:
Should probably do this in an `instanceof` check. While at the moment we're not using off road for UK, that could change in the future.